### PR TITLE
Finish size var clean up

### PR DESCRIPTION
**Pull request by Monkeychip**
**TLDR**: I am not replacing hard-coded size values. Instead, I cleaned up the remaining size vars, and moved out font-family vars into a separate utils folder.

**Longer version:** Originally, I had intended to replace all hard-coded sizes in the scss files with their corresponding size var (e.g. `width: 2.5rem` with `width: $size-2`). After some internal debate and feedback from teammates, I opted not to do this. At least not now and not in this sidebranch. While, yes, I _could_ replace a lot of hard-coded px and rem values with their respective `size_variable`, I think that work would eventually be overridden. Once we figure out our sizing vars (e.g. rem for size values and px for margins/padding??? or all rem) then we can override the hardcoded values. Any work to do so before this might be replaced. Additional, none of that needs to happen in this sidebranch in order for it to be released.

* Moved out any font-family `utils/bulma_variables` into their own file and replaced overridden values as they were not adding value.

* Replaced all `$size-small`, `$size-medium`, `$size-large`, etc. values with their respective `$size-xx` var, adding clarity to our var system.

This wraps up the size var work. There's a lot more to do, but not in the scope of this sidebranch. Next steps will be to tackle color vars.

### CODE REVIEW

File: ui/app/styles/utils/_font_family_variables.scss
Comment (Monkeychip) [inline ui/app/styles/utils/_font_family_variables.scss:1]: I could be persuaded to rename the first var `font-primary` I just went with the base var and not the one that overrode it.

File: ui/app/styles/utils/_size_variables.scss
Comment (Monkeychip) [inline ui/app/styles/utils/_size_variables.scss:1]: added comments to help add clarity on next steps and the overlap between these `size-xx` vars and the `spacing-xx` vars.

File: ui/app/styles/core/message.scss
Comment (Monkeychip) [inline ui/app/styles/core/message.scss:1]: this was my error when copying over the bulma css.

File: ui/app/styles/core/label.scss [hunk 1]
```diff
@@ -20,16 +20,6 @@
   &:not(:last-child) {
     margin-bottom: 0.25rem;
   }
-  // Sizes
-  &.is-small {
```
Comment (hellobontempo) [inline ui/app/styles/core/label.scss:24]: Out of curiosity - is the final step going through the hbs files to update the css accordingly?

File: ui/app/styles/utils/_font_family_variables.scss
Comment (hellobontempo) [reply ui/app/styles/utils/_font_family_variables.scss:1]: will there be a lot of font family stuff? it seems like this could be a more general file for `_font_variables`  and then this could later contain other font-related variables like `$font-weight-`. 

But perhaps you have another plan for those!

**Review by hellobontempo — APPROVED** [review]
Just one question! Nothing blocking

File: ui/app/styles/utils/_font_family_variables.scss
Comment (Monkeychip) [reply ui/app/styles/utils/_font_family_variables.scss:1]: That's a great idea. Let me see what I can do for my next PR.
